nixos/modules/config/console.nix

```diff
@@ -139,8 +147,13 @@ in
         environment.etc.kbd.source = "${consoleEnv pkgs.kbd}/share";
 
         boot.initrd.preLVMCommands = mkIf (!config.boot.initrd.systemd.enable) (mkBefore ''
-          kbd_mode ${if isUnicode then "-u" else "-a"} -C /dev/console
-          printf "\033%%${if isUnicode then "G" else "@"}" >> /dev/console
+          if ${isUnicode} ; then
+            kbd_mode -u -C /dev/console
```

Comment:
If only the -u changes we should keep the logic in the nix code

Reply:
`isUnicode` is now a string in bash, evaluating it to a native Nix boolean seems to require IFD.